src/Page-Components/Article.jsx

```diff
@@ -6,12 +6,19 @@ import "../css/Style.css";
 import "../css/Article.css";
 import Loading from "./Loading";
 import fetchApiData from "../api";
+import CommentCard from "../Components/CommentCard";
 
 const Article = () => {
+  const { article_id } = useParams();
   const [article, setArticle] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
+  const [comments, setComments] = useState([])
 
-  const { article_id } = useParams();
+  useEffect(() => {
+      fetchApiData.get(`/articles/${article_id}/comments`)
+      .then((comments) => {setComments(comments.data.article_comments)})
+      setIsLoading(false)
```

Comment:
This isLoading state should be in the `.then` block, otherwise it'll be set to false before the data's been loaded